### DISCUSSION TITLE
What is the difference between Obtainium and IzzyOnDroid?

### DISCUSSION THREAD

**Discussion by HFPasfho** [ImranR98/Obtainium#1589] [Q&A]
At a high level, both solutions seem to do the same thing, which is to take updates straight from the source from the developers. So far I've only found one program whose creators are incredibly stubborn about not publishing their program in F-Droid or IzzyOnDroid or anywhere other than Obtainium, and that's the only difference I've noticed, but there's definitely something I don't know about Obtainium. How much additional value does Obtainium provide compared to IzzyOnDroid?

**Comment by Mikaela** [comment]
Obtainium downloads apk files from wherever you tell it to and you curate it.

IzzyOnDroid is an F-Droid repository and curated by Izzy.

**Reply by HFPasfho** [reply]
So IzzyOnDroid is actually better because there is someone who verifies the applications whether they comply with security and privacy standards? Do I understand correctly?
I remember when Simple Mobile Tools was secretly sold to some shady corporation and the update contained ads and spy scripts. Then F-Droid protected its users from the malicious update and informed them about the threat. I can't count on anything like that in Obtainium, right?

**Reply by Mikaela** [reply]
When you install Obtainium, the only app available is Obtainium itself for keeping up-to-date. You are responsible for adding the apps you want to it and no one else is deciding what you want or don't want to install.

Remember that while F-Droid does have automated tests, it's not perfect either and neither is Play Store or any other app store. The final responsibility is on you.

**Reply by ImranR98** [reply]
Yep, IzzyDroid is a repo of apps, Obtainium is an automation tool - apples and oranges.
